haskell-indexer-backend-ghc/src/Language/Haskell/Indexer/Backend/Ghc.hs

```diff
@@ -506,7 +507,29 @@ refsFromRenamed ctx declAlts (hsGroup, _, _, _) =
         TypeSig names _ _ ->
 #endif
             mapMaybe (\(L l n) -> give ctx (nameLocToRef n TypeDecl l)) names
+        (PatSynSig lnames _) ->
+          lkupSigNames [lnames]
+        (FixSig (FixitySig lnames _)) ->
```

Comment:
If these are cases which were not yet crosslinked, could you also update the tests? At least the GHC backend tests at https://github.com/google/haskell-indexer/blob/master/haskell-indexer-backend-ghc/tests/Language/Haskell/Indexer/Backend/Ghc/Test/TypeLinkTestBase.hs, and optionally the GHC+Kythe end-to-end tests - see https://github.com/google/haskell-indexer/tree/master/kythe-verification/testdata/basic, though for GHC+Kythe the type-link tests are not yet ported.